lab/lab1/part4-project-euler.html

```diff
@@ -32,6 +32,53 @@
 
     ===================== */
 
+    //Probelm 1
+
+    var num = [];
+    var i ;
+    for (i = 0; i < 1000; i ++) {
+      if (i % 3 === 0 || i % 5 === 0 && i < 1000) {
+        num.push(i)
+      }
+    }
+
+
+    var result = num.reduce((accumulator, currentValue) => {
+      return accumulator + currentValue
+    })
```

Comment:
I think you got the right answer... but it's hard to tell based on the console log

![image](https://user-images.githubusercontent.com/8103418/52544111-a8a39c00-2d7c-11e9-9665-b5c89dbd3080.png)